extras/usd/tutorials/endToEnd/tutorial_scripts/add_shadingVariants.py

```diff
@@ -93,6 +93,7 @@ def _AddShadingToBall(stage):
         with shadingVariant.GetVariantEditContext():
             whichBall = variantName.split('_')[-1]
             texPath = os.path.join(texDir, 'ball%s.tex' % whichBall)
+            texPath = texPath.replace('\\', '/')
             # in the current variant, modify the color
```

Comment:
I guess yeah. It's a tutorial so maybe it's not that important, but I was thinking that it's probably good that normalization is consistent everywhere, especially if the rules ever get more complex. For example, at ILM, we had scripts that performed much more stringent normalization including illegal character substitutions. If it all goes through one place, that's enabled universally.